edc-extensions/hashicorp-vault/src/main/java/org/eclipse/tractusx/edc/hashicorpvault/AbstractHashicorpVaultExtension.java

```diff
@@ -50,6 +51,14 @@ public class AbstractHashicorpVaultExtension {
 
     private static final String VAULT_TIMEOUT_SECONDS = "edc.vault.hashicorp.timeout.seconds";
 
+    public HashicorpVaultClient createVaultClient(ServiceExtensionContext context, ObjectMapper mapper) {
+        var config = loadHashicorpVaultClientConfig(context);
+
+        final OkHttpClient okHttpClient = createOkHttpClient(config);
```

Comment:
avoid final variables, use `var`

Reply:
i would have suggested the same :) 
I did refactor the code and forgot to chop the final :D